src/Tabs/tabs.css

```diff
@@ -0,0 +1,102 @@
+.d2ui-tabs {
+    box-shadow: 0px 2px 4px -1px rgba(0, 0, 0, 0.2), 0px 4px 5px 0px rgba(0, 0, 0, 0.14),
+        0px 1px 10px 0px rgba(0, 0, 0, 0.12);
+    width: 100%;
+    display: flex;
+    z-index: 1100;
+    box-sizing: border-box;
+    flex-shrink: 0;
+    flex-direction: row;
+    background-color: #eceff1;
+
+    &--relative {
+        position: relative;
+    }
+
+    &--fixed {
+        position: fixed;
+        top: 0;
+        left: 0;
```

Comment:
Sticky (Line 29) and Fixed (Line 16) positions should have higher `z-index` than the inline Tabs (Line 6). Not sure why all tabs need `z-index: 1100`?

Reply:
I removed the z-index for the generic tabs, and put a z-index on fixed, absolute and sticky. Fixed has the highest z-index because the others should scroll underneath this one.